src/main/resources/twilio-go/api_doc.mustache

```diff
@@ -14,11 +14,45 @@ Method | HTTP request | Description
 
 ## {{{operationId}}}
 
-> {{#returnType}}{{{returnType}}} {{/returnType}}{{{operationId}}}(ctx, {{#allParams}}{{#required}}{{paramName}}{{#hasMore}}, {{/hasMore}}{{/required}}{{/allParams}}{{#hasOptionalParams}}optional{{/hasOptionalParams}})
+> {{#returnType}}{{{.}}} {{/returnType}}{{{operationId}}}(ctx{{#pathParams}}, {{paramName}}{{/pathParams}}){{#allParams}}{{^isPathParam}}.{{vendorExtensions.x-export-param-name}}({{paramName}}){{/isPathParam}}{{/allParams}}.Execute()
 
 {{{summary}}}{{#notes}}
 
-{{{notes}}}{{/notes}}
+{{{unespacedNotes}}}{{/notes}}
+
+### Example
```

Comment:
We have a [jira](https://issues.corp.twilio.com/browse/DI-1232) for this. I don't think the generated code examples our customizations.